mithril-aggregator/src/runtime/runner.rs

```diff
@@ -232,7 +233,7 @@ impl AggregatorRunnerTrait for AggregatorRunner {
     }
 
     async fn is_certificate_chain_valid(&self, time_point: &TimePoint) -> StdResult<()> {
-        debug!(self.logger, "RUNNER: is_certificate_chain_valid");
+        debug!(self.logger, ">> is_certificate_chain_valid");
```

Comment:
Would it be useful to add the timepoint?
```suggestion
        debug!(self.logger, ">> is_certificate_chain_valid"; "time_point" => #?time_point);
```

Reply:
I don't think that in this specific case this would more add noise than be useful because validating the certificate chain is not related to the current timepoint.